USAGE_android.md

```diff
@@ -413,8 +773,235 @@ Key(s) | Key code(s) | Action
+`/sdcard'.
+**NOTE**: It actually lists the full file in the message like the following:
+
+```text
+gfxrecon: Recording graphics API capture to /sdcard/gfxrecon_capture_frames_500_through_700_20221211T130328.gfxr
+```
+
+But, let's say you didn't know, so double check that folder:
+
+```bash
+adb shell ls /sdcard
+```
+
+The output might look like this:
+
+```text
+Alarms
+Android
+Audiobooks
+DCIM
+Documents
+Download
+Movies
+Music
+Notifications
+Pictures
+Podcasts
+Recordings
+Ringtones
+boot.img
+com.khronos.vulkan_samples
+gfxrecon_capture_frames_500_through_700_20221211T130328.gfxr
+libVkLayer_gfxreconstruct.so
+libVkLayer_khronos_validation.so
+libc++_shared.so
+```
+
+There's our file right there.
+
+### 8. Install the Replay Application
+
+Now, we need to install the Replay application that we built as part of the
+GfxReconstruct source.
+
+First, let's make sure we don't have an old version lying around:
+
+```bash
+adb uninstall replay-debug.apk
+```
+
+Then, from the top of the source pulled down from the repo by using the
+`gfxrecon.py` script:
+
+```bash
+./android/scripts/gfxrecon.py install-apk android/tools/replay/build/outputs/apk/debug/replay-debug.apk
+```
+
+### 9. Run the Replay
+
+Try running the replay using the `gfxrecon.py` script:
+
+```bash
+./android/scripts/gfxrecon.py replay /sdcard/gfxrecon_capture_frames_500_through_700_20221211T130328.gfxr
+```
+
+However, it fails.
+Looking at the `adb logcat` output we see the following:
+
+```text
+gfxrecon: API call vkCreateDevice returned error value VK_ERROR_EXTENSION_NOT_PRESENT that does not match the result from the capture file: VK_SUCCESS.  Replay cannot continue.
+```
+
+Let's re-try again, but removing unsupported extensions:
+
 
-Virtual Swapchain insulates higher layers in the Vulkan stack from these problems by creating a set of images, exactly matching the swapchain configuration at capture time, which it exposes for them to render into.  Before a present, it copies the virtual image to a target swapchain image for display. Since this issue can happen in many situations, virtual swapchain is the default setup. If the user wants to bypass the feature and use the captured indices to present directly on the swapchain of the replay implementation, they should add the `--use-captured-swapchain-indices` option when invoking `gfxrecon-replay`.
+```bash
+./android/scripts/gfxrecon.py replay --remove-unsupported /sdcard/gfxrecon_capture_frames_500_through_700_20221211T130328.gfxr
+```
+
+Voila!
+It worked, the replay executes properly.
+
+### 10. Cleanup
+
+Make sure once you're done, you disable the layer or else every time you run
+you will be performing a new capture:
+
+```bash
+adb shell settings put global enable_gpu_debug_layers 0
+```
+
+Also, if you need to share your replay file, make sure to pull it off of the
+device for sharing:
+
+```bash
+adb pull /sdcard/gfxrecon_capture_frames_500_through_700_20221211T130328.gfxr
+```
```

Comment:
I didn't see anything about disabling the layer when done, so I added it to my example.